src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -150,6 +151,42 @@ public void testFromJenkinsValidProxyInfoJenkins(String url, List<Pattern> ignor
         validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, proxyUsername, proxyPassword, testNtlmDomain, StringUtils.EMPTY);
     }
 
+    @ParameterizedTest
+    @MethodSource({ "validProxyInfo"})
+    public void testFromJenkinsValidProxyInfoNullUserName(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
+        ProxyConfiguration proxyConfigurationMock = Mockito.mock(ProxyConfiguration.class);
+        Jenkins jenkinsMock = Mockito.mock(Jenkins.class);
+        JenkinsWrapper jenkinsWrapper = new JenkinsWrapper(jenkinsMock);
+
+        setJenkinsFields(jenkinsMock, proxyConfigurationMock);
+
+        Mockito.when(proxyConfigurationMock.getUserName()).thenReturn(null);
+        Mockito.when(proxyConfigurationMock.getPassword()).thenReturn(StringUtils.EMPTY);
+        Mockito.when(proxyConfigurationMock.getNoProxyHostPatterns()).thenReturn(ignoredProxyHosts);
+
+        ProxyInfo proxyInfo = JenkinsProxyHelper.fromJenkins(jenkinsWrapper).getProxyInfo(url);
+        validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, null, StringUtils.EMPTY, testNtlmDomain, StringUtils.EMPTY);
+    }
+
+    @ParameterizedTest
+    @MethodSource({ "validProxyInfo"})
+    public void testFromJenkinsValidProxyInfoNullNtlmDomain(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
+        String returnProxyUserName = "\\" + proxyUsername;
```

Comment:
In all other valid tests, the proxyUserName was adjusted to be testNtlmDomain\proxyUsername **IF** testNtlmDomain contained a valid String. This was flagged by Jacoco branch coverage, so this tests for an empty ntlmDomain, but the UserName still beginning with a '\'.

Reply:
In review with @rottebds, it was pointed out that better use of input method params could be implemented which would add the coverage that this test was specifically doing. This test has been removed.